flake.nix

```diff
@@ -10,6 +10,8 @@
     shared-kernel.inputs.nixpkgs.follows = "nixpkgs";
     beckn-gateway.url = "github:nammayatri/beckn-gateway";
     beckn-gateway.inputs.shared-kernel.follows = "shared-kernel";
+    easy-purescript-nix.url = "github:justinwoo/easy-purescript-nix";
+    easy-purescript-nix.flake = false;
```

Comment:
Group these out with a `# Frontend inputs` comment (just like the backend inputs).